packages/api/src/auth/jwt/jwt.mapper.ts

```diff
@@ -0,0 +1,15 @@
+import { JwtPayload, UserFromJwt } from './jwt.types';
+
+export class JwtMapper {
+  static fromDomainToJwt(user: UserFromJwt): JwtPayload {
+    return {
+      sub: String(user.id),
+    };
+  }
+
+  static fromJwtToDomain(payload: JwtPayload): UserFromJwt {
+    return {
+      id: Number(payload.sub),
```

Comment:
It's better to use string for ids in domain. It leaves you flexibility if its a number / uuid or some natural key (like email).